### DISCUSSION TITLE
Security Hub integration

### DISCUSSION THREAD

**Discussion by qadri99-max** [prowler-cloud/prowler#4820] [Q&A] [answered]
Hello, few questions about Prowler with Security Hub:

Once Prowler is integrated with Security Hub:
a - Are there extra charges for this integration?
b - Does Prowler save my account details or findings within their own servers?

If yes to both of the above questions, how else can I save findings into Security Hub? 
I am saving the findings as ASFF json format.

thanks

**Comment by pedrooot** [comment, marked as answer]
Hi! @qadri99-max you can find the full info about Prowler Security Hub integration [on Prowler docs](https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/aws/securityhub/), regarding to your question:
a - You can check SH pricing [here](https://aws.amazon.com/security-hub/pricing/), enabling this integration will incur in costs in AWS Security Hub.
b - No, take into account that Prowler runs in your environment so we don't get any info from your execution (account, findings...)

**Reply by qadri99-max** [reply]
@pedrooot Thanks for the reply.

Under Security Hub's Prowler integration details, there is one line that is 
confusing for me. It says: "Purchase a subscription to this product". 
I thought it meant there is a Prowler charge on top of Security Hub charges.

So can I confirm that if I am running Prowler from my own Codebuild project, and enable Prowler integration, nothing goes out of my account?

thanks

<img width="1106" alt="enable-partner-integration" src="https://github.com/user-attachments/assets/5e220136-333d-4a80-94c4-ef6d32b0e55e">

**Reply by pedrooot** [reply]
Yes! No info related with your account will go out

**Comment by toniblyx** [comment]
Hello! if you see all products in the integrations section, they all have "purchase" option, that is something we can´t change but of course you can use that integration with Prowler from the command line (free) or purchasing a subscription in our SaaS offering. In both cases the integration works the same and enabling it in the SH side is the same.

![Screenshot 2024-08-22 at 13 20 46](https://github.com/user-attachments/assets/eb5101a2-cdf5-4ca8-9683-350431f936bf)

**Reply by qadri99-max** [reply]
thanks @toniblyx
